### DISCUSSION TITLE
Themed Events

### DISCUSSION THREAD

**Discussion by VV-LX** [ppy/osu#28841] [Ideas]
Beatmap Spotlights tried to give a focus for players and reward mappers with player attention. However, they didn’t result in a noticeable, lasting effect.

###Events
- Providing [osu!Coins](https://github.com/ppy/osu/discussions/28837) for completing Spotlight Maps would also give good reason to play them during their season, effectively raising playcounts for these maps

-The generic Autumn Spotlight Silver 1 badge is too basic and uninteresting. Something as tailored as a tournament banner should be genuinely appealing and stand out (I'd say something similar about Medals)

- Events with big rewards (such as cosmetics, badges) and a live competitive element could require Entry Tickets, purchased with [osu!Coins](https://github.com/ppy/osu/discussions/28837). Having several methods to both gain and spend [Coins](https://github.com/ppy/osu/discussions/28837) is necessary to maintain their perceived value

- Letting players set up [Coin](https://github.com/ppy/osu/discussions/28837) Bounties as part of winning Multiplayer Matches or Modding could help increase their usage and circulation

Events could also be done in collaboration with (featured) artists or IP's as promotion activities. 

It could also have an easy tie in with osu!' Story Mode #22270. Having an Event for new content releases in general would drive engagement.

**Comment by bdach** [comment]
https://github.com/ppy/osu/discussions/27983 is coming.

not sure about the whole idea of gating stuff behind 'coins'. feels like a bad mobile game mechanic.

**Reply by VV-LX** [reply]
Plenty of games use one type of currency or resource to manage pacing. I think the difference lies in whether it's used to manipulate the player into spending more time/money than they normally would. "Coins" themselves are only a mechanic for the sake of convenience. You could skip that step and have Level milestones/Medals Achievements give cool cosmetics and rewards. The point is cosmetic features feel underutilized and underdeveloped when they could be effective in promotion, for stuff like Loved and Spotlight.

**Reply by Stoppedpuma** [reply]
I have to agree with bdach on this one, coins just feel really shitty to me. Especially when this gates "events", I feel this really makes osu! feel more like a "cash grabby" free to play game.

> Events with big rewards

This kind of ties into the [other discussion you made](28837) but I don't think "big rewards" is a good idea since it takes away from the simplistic feel of osu!. Small things like profile covers maybe, but leading into nearly everything else you mentioned in that discussion under "Cosmetics" and "Coin Shop" definitely doubles down on the "cash grabby" free to play feel.

**Reply by VV-LX** [reply]
### The choice to use language like "Coins" and "Shop" likely presented a completely different image as to what I had in my head.

I imagined logging onto the game and seeing something like the Spotlights in essence but expanded. There would be temporary Quests available specific to the beatmap (FC this stream/jump/chorus Section of the beatmap, get above a certain total acc once). 

Completing a Quest would give currency which would be used to buy a cosmetic (like a token or icon to commemorate the event). **There would be no way to progress by spending money.**

**You could also forgo an Event Currency/Shop** and simply give Rewards directly for fulfilling certain criteria. I only thought of it this way because that's how most other games I've played do it. There's probably a game design idea embedded in that decision, but I wouldn't know.

The cosmetics would consist of **user-submitted** content, like the Art Contests, which have grown higher and higher in quality.
The Shop would be designed with having the Cosmetics be as obtainable as possible for **all intended players**. 

Having them feel valuable for progressively more skilled players can be done in other ways without having to make the entire event unapproachable for newer/casual players.

What was in my head didn't make me think the events/coins would prevent players from earning commemoratives. In fact, it would give players multiple ways to earn rewards. 

Say it's just like Spotlight (the gameplay loop would end up being similar). There's an even spread of difficulties with multiple maps per difficulty selected (Hard, Insane, Expert, etc.) Each difficulty has one-time and repeatable quests. Assuming you **can't FC** maps in each category and the combined Drain Time being 1hr:

- Hard map/missions would take, say, 1-3 hours of exclusively playing them. If you could, say, FC all of the Hard diffs with 99 acc FC on sightread, it would take as long as Drain Time or less (using Mods to complete multiple missions in one run)

- Insane Missions could be completed to give enough Currency to max out the Shop in around 1-2hrs of only these quests.

- Expert Missions would max out the shop in under an hour.

If my numbers sound strange, they're pretty random. It can all be scaled by adjusting how much Currency gets rewarded if you're concerned about alienating groups of players.

Most could play every map once, get the Map Clear/Miscellaneous Mission Rewards and max out the Shop after going through every map.

I don't think making content time-gated is an isolated negative but the other side of the emotional coin that can make these Rewards feel more valuable, making the effort to get them a more engaging experience. You could create two shops, an Event Shop and a Permanent Shop, so after an Event is over all the cosmetics would be moved to the Permanent Shop and could be obtained through buying Supporter, the Event itself being saved in an in-game Archive where you can replay it solo/multi.

A lot of the big, competitive games that use this template get negative (or at best neutral) reaction. I think it's because they're forced to make certain decisions that are because they're more oriented towards profit over player experience. Since osu doesn't have that interest, it can use this idea without making decisions that'll cause players to feel like that. The best of both worlds.


### Simple
![image](https://github.com/user-attachments/assets/1fd683a7-d1dc-40f2-947a-0e8d3f101bd4)

![image](https://github.com/user-attachments/assets/de2e3cb5-6a4f-4c65-875c-adf197f6f129)



I haven't accounted for how every player would view these changes or how much they value osu's _current_ simplicity.

There probably was a "simple" draw to it, but I think that's been gone since the first Icon change. Every year we get a larger community, more complex mapping ideas, more ambitious events (The Roundtable being the first LAN tournament for osu in **2022**)

I think the feeling of community this game gives is what keeps people.

The ones I know don't stay with the game because of gameplay (every rework brings its share of complaints), or how something looks in the UI, or because of how many gamemodes there are. They stay because they find value in being a part of the osu community. 

BN's do their work not out of obligation but because it feels meaningful to push maps into Ranked and give mappers that sense of achievement. 
The top level players push their limits and go for records because of how the community elevates that achievement.


With that perspective, I don't think 2009 osu! and current osu! stable are both the same "simple", and if osu!Lazer's miscellanea of mods and sleek UI are in line with simple, a couple new places to put cool pictures on your profile won't change that.

**Reply by VV-LX** [reply]
I feel a bit bad for writing this much. Also, the "Big Rewards' I had in mind were "custom rewards". If back in 2016 cookiezi got a cosmetic to commemorate his HDHR Freedom Dive score. Or for Karcher's Yomi Yori FC. "Big" because they would formalize grand achievements in-game.

**Reply by Stoppedpuma** [reply]
>The choice to use language like "Coins" and "Shop" likely presented a completely different image as to what I had in my head.

One of the fail points of modern games I find is pressure on players for things like limited time events that give you exclusive items which will not return. I view this as a failure of game design and "predatory" as this makes people want to spend unnecessary amounts of time or in most cases money to get something assuming they don't miss it all together. I disagree with your statement of "obtained through buying supporter" as this also feels like gimmick to make users spend money.

> the Event itself being saved in an in-game Archive where you can replay it solo/multi.

Playlists are already saved in the "Participated" tab.

![image](https://github.com/user-attachments/assets/0db7fd0d-d0b7-4bf5-996d-e03f342d1c2e)


> Simple

I guess a better way to explain simple in the case of osu! would be "not in your face or flashy"? Simple makes sense to me in comparison to other applications. Take discord for example, no clue why it has 700 different cosmetics and various other things which I think are in the way visually, especially for a *messaging* application. In comparison,  osu! is kept in a more readable flat design which I think is more desirable from a design perspective over some of the nightmares I've seen outside of osu!.

> Big Rewards

These are usually given out in the form of user titles or badges and I think is a great way since it's not obtrusive to the rest of the design.

**Reply by VV-LX** [reply]
> exclusive items which will not return

In essence, what I wanted was for more Events in osu with a specific Theme (like only maps with from handful of artists, source, skillset) and have rewards provided that were relevant to that Theme (Banners, Icons, etc).

Quests could also give more defined, intentional path to progress and get rewarded instead of relying on player's to enjoy the maps/feel satisfied with a LB rank.

I thought these Events would be recurring, coming back every few weeks or months.

> this also feels like gimmick to make users spend money.

Gotcha, I thought it could add more value to Supporter rather than hurt people who didn't buy it. I think Cosmetics are tame way to give a reason to spend money to support a game, since unlike DLCs/Game Items, they don't provide in-game advantages so it's easier to ignore them. Along with the fact Events are recurring, it would just be a matter of whether you can wait to play the Event again for the cosmetics or want them now.

**Reply by Stoppedpuma** [reply]
> rewards provided that were relevant to that Theme (Banners, Icons, etc).

Covers would be the most likely, icons at most but both of these are bypassable by simply saving the image.

> I think Cosmetics are tame way to give a reason to spend money to support a game.

Supporter has always been more of a QOL thing then a locked wall of cosmetics that other games seem to do. Additionally anything that's paid for cosmetics has been a 99% no from both player and developer opinions as well as peppy even going out of his way to say no about a battle pass like thing a few years back on twitter.

> Along with the fact Events are recurring

At the moment official events are skill based and award upon it. Spotlights give permanent badges to the top 1%. I don't really see a need for anything more as it's a fun way to find new maps and possibly be rewarded from it.

**Reply by VV-LX** [reply]
> Spotlights give permanent badges to the top 1%.

I think Spotlights/Events would be more appealing if there were Rewards for more participants. That makes it feel like the Event doesn't really care about the other 99% who participate/score high, just not in the top percent.

**Reply by Stoppedpuma** [reply]
The entire point of badges is for doing something worthy of one, they aren't participation trophies.

**Reply by VV-LX** [reply]
I absolutely agree, even at the risk of unironically sounding like that "ruining the rarity of X" meme. 

I'd like osu to have more definable milestones however, because currently, the game offers two main sources of positive feedback to playing the game: Rank and Improving. Both are powerful, but I've felt burnt out chasing after them. While we could say "Step away from the game" I feel like there's also the option to simultaneously add more ways to enjoy the game. 

If the devs want to keep Badges exclusively a "high achievement" honor, couldn't we add other "participation trophies"? I feel like you say that with a negative implication, but people only dislike participation awards if they're given without thought/feel useless. I don't think most people would turn down a Cookie or $20 for just participating in something. The reason most don't do either is it'd be too expensive, but I don't see that "cost" if osu! were to implement a way to cosmeticize, then provide cosmetics.

I don't think I'm going against the current by saying this. Initiatives like Reworking Levels and Story Mode are both clearly moves in the same direction I'm pointing, but, I think giving Events a wider range of reasons to play would lead them to appeal to both highly skilled players and not as skilled players.

**Reply by Squiddu** [reply]
Problem there I think is more the idea that either you give stuff to the 1% or literally everyone, which I don't think that's what they were going for.

Games like Splatoon 3 have decent events that give out tiers of reward for the top 5%, 25% and 50%, something along those numbers. Of course for a game like osu! I don't think letting it reach as far as 50% is a good idea, but the playerbase has such a huge range of skill levels in different ways that it does seem discouraging to say 99% of players basically have no reason to participate beyond the goals they put on themselves. 

I get osu! putting emphasis on this aspect of self-improvement, but this specifically feels like the absolute extreme of that idea.

**Reply by Stoppedpuma** [reply]
> 1%

Small correction here, it seems to be the top 2% but I misremembered it as 1%, my bad.

> but the playerbase has such a huge range of skill levels in different ways that it does seem discouraging to say 99% of players basically have no reason to participate beyond the goals they put on themselves.

This isn't necessarily true, beatmap spotlight do actually still award people below the top 2% just the badges given are temporary instead of permanent.

![image](https://github.com/user-attachments/assets/be88a987-c39e-4ccf-9343-d95fe379d3eb)

**Reply by VV-LX** [reply]
I'm not fixated on Badges as a reward. I'd like other kinds of Rewards besides badges. 

I think far more people would play Spotlights if they had that little incentive. Having temporary badges sets it to be just like Rank; a grind to defend and maintain your position.

**Reply by Stoppedpuma** [reply]
There was an argument a few days ago in the dev discord about something similar to this (rewards) being involved in the daily challenge, there was even of people against it to the point of not even wanting daily challenge to exist so I'm going to say if anything this will cause even more problems.

> I think far more people would play Spotlights if they had that little incentive.

Before lazer had ranked leaderboards, the majority of interested stable players would play lazer for spotlights alone. Basing spotlights popularity off numbers which didn't have a lot of players due to other factors is an irrelevant argument. A better example would be to look at daily challenge where around 6k players participated IIRC.
